scripts/policy/frameworks/cluster/experimental.zeek

```diff
@@ -0,0 +1,85 @@
+##! Experimental features of the Cluster framework.
+
+@load base/frameworks/cluster
+
+module Cluster::Experimental;
+
+export {
+	## When using broker-enabled cluster framework, this event will be sent to
+	## the manager once a cluster node has set up all its outgoing connections
+	## to other cluster nodes based on the given cluster layout.
+	##
+	## name: The name of the now fully connected node.
+	##
+	## id: The identifier of the now fully connected node.
+	##
+	## resent: If true, the node has previously signaled that it is fully
+	##         connected. This may happen in case the manager restarts.
+	##
+	## .. warning::
+	##
+	##     There is no tracking of cluster node connectivity. Thus, there is
+	##     no guarantee that all peerings still exist at the time of this event
+	##     being raised.
+	global node_fully_connected: event(name: string, id: string, resent: bool);
```

Comment:
This is suuuper picky, but "resent" suggests something else to me, that happened in the past, as opposed to flagging that it is this event that is being re-sent. It also means to despise. :-) How about "resending", "repeat" or "retransmit"?